STM32Cube/Tasks/millis.h

```diff
@@ -9,6 +9,5 @@
 #define MILLIS_H_
 
 float millis(void);
```

Comment:
For some reason we now have a conflict between this `millis()` function and one defined in `PIC18F26K83_timer.h`. That file also has a function called `millis()` (which is a UINT) and thats causing a conflict, not sure what the best way to resolve this is, C not having stricter file scopes like python is still wild for me to this day.